### PR TITLE
Update Create and Add Feature in UG

### LINKED ISSUE #145: Update Create Feature in UG

### PR DESCRIPTION
**Pull request by jicsontoh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -47,9 +47,9 @@ interactions recorded faster and simpler than traditional GUI applications.
 
    * **`list`** : Lists all companies.
 
-   * **`create`**`n/James coy/MacDonalds p/98765432 e/jamesho@example.com` : Creates a point-of-contact named `James` linked to the Company `MacDonalds`.
+   * **`create`**`create 2 n/James p/97814456 e/test@test.com t/MainPoc t/owesMoney` : Creates a point-of-contact named `James` linked to the company at index 2.
 
-   * **`delete`**`coy/MacDonalds` : Deletes MacDonalds (Company) together with all their contacts in the current list.
+   * **`delete`**`delete 1` : Deletes the company at the first index together with all their contacts in the current list.
```
Comment (ElginL) [inline docs/UserGuide.md]: Should we also state that all the transactions tagged to the company is also deleted?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -91,13 +91,15 @@ interactions recorded faster and simpler than traditional GUI applications.
...
+Format: `add n/NAME a/ADDRESS [t/TAG]...`
 
+* Tags are optional.
+* Multiple tags can be tagged to the company.
 * Use `create` command to add POC to the company.
 
 Examples:
-* `add n/MacDonalds a/West Coast Park`
-* `add n/KFC a/Yishun Street 81`
+* `add n/MacDonalds a/West Coast Park `
```
Comment (Jo3LW) [inline docs/UserGuide.md]: Should we standardise and remove the space behind Park?

**Review by ElginL — APPROVED** [review]
LGTM, except for one possible line change 👍

**Review by Jo3LW — APPROVED** [review]
LGTM

**Review by Jo3LW — APPROVED** [review]
LGTM

**Review by notle1706 — APPROVED** [review]
LGTM

**Pull request merged**